### PR TITLE
Fix JS NPE when viewing specific range of PR commits

### PR DESCRIPTION
**Pull request by delvh**
This should be the easiest fix.
While other solutions might be possible that exterminate the root cause, they will not be as trivial.

### CODE REVIEW

File: web_src/js/components/DiffCommitSelector.vue [hunk 1]
```diff
@@ -94,7 +94,8 @@ export default {
     focusElem(elem, prevElem) {
       if (elem) {
         elem.tabIndex = 0;
-        prevElem.tabIndex = -1;
+        if (prevElem)
+          prevElem.tabIndex = -1;
```
Comment (silverwind) [inline web_src/js/components/DiffCommitSelector.vue]:
```suggestion
        if (prevElem) prevElem.tabIndex = -1;
```

I prefer either single-line if with no brackets or multi-line with brackets.

**Comment by silverwind** [comment]
NPE surely isn't the right term? JavaScript does not have pointers 😆

**Review by silverwind — APPROVED** [review]

**Comment by delvh** [comment]
Technically, JS calls it `TypeError`.
But still, it is basically a `NullPointerException`, simply as that's the term that everyone knows.
Java has no explicit pointers either, and still it's a `NullPointerException` there… :grin:

**Comment by delvh** [comment]
Update: Okay, I just found another bug in the same functionality:
Whenever I select a single commit in that menu, it shows `showing only changes of commit <non-existing commit SHA>`.
The URL is correct, however.

**Comment by delvh** [comment]
@sebastian-sauer do you have any idea perhaps what might be causing `↑`?

**Comment by sebastian-sauer** [comment]
I'll try to have a look into that.

**Comment by sebastian-sauer** [comment]
Looks like the reference to BeforeCommitID is wrong in the tmpl https://github.com/go-gitea/gitea/blob/main/templates/repo/diff/box.tmpl#L56

When only one commit is shown this should use AfterCommitID

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**